.github/workflows/monthly_release.yml

```
@@ -0,0 +1,39 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"  # every first day of the month at midnight
+
+jobs:
+  monthly_release:
+    name: Monthly release
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Configure Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+      - name: Create new branch
+        run: |
+          git checkout -b monthly_release
```

Checkout an existing branch, if any:
```suggestion
          git checkout monthly_release || git checkout -b monthly_release
```